### PR TITLE
Add amp-ad support for Ad Generation

### PR DESCRIPTION
**Pull request by adg-dev**
We added "adgeneration" that the type of amp-ad.
Would you merge this commit?

We already signed a CLA.

- CLA name: adg-dev
- Corporation Name: Supership Inc.

### CODE REVIEW

File: ads/_config.js [hunk 1]
```diff
@@ -169,6 +170,7 @@ export const adPreconnect = {
     'https://output.nend.net',
     'https://img1.nend.net',
   ],
+  adgeneration: 'https://i.socdm.com',
```
Comment (cramforce) [inline ads/_config.js]: No need to repeat here, if it is the same hostname as above.

File: ads/adgeneration.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+/**
+ * URL encoding of query string
+ * @param {!String} str
+ */
+function encodeQueryValue(str) {
+  return str.split('&').map(v => {
+    const key = v.split('=')[0],
+      val = v.split('=')[1];
+    return key + '=' + encodeURIComponent(val);
```
Comment (cramforce) [inline ads/adgeneration.js]: encode `key` as well?

**Comment by cramforce** [comment]
Looks great. One comment.

**Comment by adg-dev** [comment]
@cramforce Thank you for review. We've fixed them.

**Comment by cramforce** [comment]
LGTM

**Pull request merged**